src/NosCore.Database/DAL/ItemInstanceDAO.cs

```diff
@@ -35,88 +39,80 @@
 
 namespace NosCore.Database
 {
-    public class ItemInstanceDao : IGenericDao<IItemInstanceDto>
+    public class ItemInstanceDao : IDao<IItemInstanceDto?, Guid>
     {
         private readonly ILogger _logger;
-        private readonly PropertyInfo? _primaryKey;
+        private readonly PropertyInfo[] _primaryKey;
+        private readonly IDbContextBuilder _dbContextBuilder;
 
-        public ItemInstanceDao(ILogger logger)
+        public ItemInstanceDao(ILogger logger, IDbContextBuilder dbContextBuilder)
         {
             _logger = logger;
+            _dbContextBuilder = dbContextBuilder;
+            using var context = _dbContextBuilder.CreateContext();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'context'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)